libr/core/cmd_anal.c

```diff
@@ -3873,6 +3866,7 @@ static int cmd_anal_all(RCore *core, const char *input) {
 		"aat", " [len]", "analyze all consecutive functions in section",
 		"aap", "", "find and analyze function preludes",
 		"aav", " [sat]", "find values referencing a specific section or map",
+		"aau", " [len]", "list mem areas (larger than len bytes) not covered by functions",
```

Comment:
There are no tests for ai so we can change that behaviour withiut breaking travis. We can make current ai keep doing this and add subcommands for your stuff

> On 20 Apr 2016, at 21:01, Davide Balzarotti <notifications@github.com> wrote:
> 
> In libr/core/cmd_anal.c:
> 
> > @@ -3873,6 +3866,7 @@ static int cmd_anal_all(RCore *core, const char *input) {
> >  		"aat", " [len]", "analyze all consecutive functions in section",
> >  		"aap", "", "find and analyze function preludes",
> >  		"aav", " [sat]", "find values referencing a specific section or map",
> > +		"aau", " [len]", "list mem areas (larger than len bytes) not covered by functions",
> I agree.. 
> but problem is, now ai is used to print information about an address (page permissions, etc.)
> which btw should probably not be under a since it is not related to analysis.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>